### PR TITLE
Update domains

### PR DESCRIPTION
**Pull request by evanwchurchill**
**Ticket(s):** https://app.clubhouse.io/greenspace/story/36932/update-wtc-sso-integration-guide-and-sample-oidc-client

### Summary

This PR updates the staging URLs, as well as flattens the directory structure, removing the `oidc` dir

### Changes

- Update staging URLs
- Flattens directory structure

### CODE REVIEW

File: oidc-client-sample.js [hunk 1]
```diff
@@ -15,14 +15,14 @@ document
 Oidc.Log.logger = console;
 Oidc.Log.level = Oidc.Log.INFO;
 
-var staging = 'https://app.wellnesstogether.grnspace.co/oauth2';
+var staging = 'https://app.wellnesstogether.grnspace.ca/oauth2';
 var prod = 'https://ca.portal.gs/oauth2';
```
Comment (matthewshirley) [inline oidc-client-sample.js]: `ca.portal.gs` is old. 🙈

**Review by matthewshirley — APPROVED** [review]
Thanks, @evanwchurchill. LGTM! 🐗

File: oidc-client-sample.js (hunk 1 above)
Comment (evanwchurchill) [reply oidc-client-sample.js]: nice find!

File: index.html [hunk 2]
```diff
@@ -226,7 +226,7 @@ <h3>Without JS</h3>
 
     <!-- WTC Signin JS for rendering that button -->
     <script
-      src="https://cdn-ca.grnspace.co/wtc_signin.min.js"
+      src="https://cdn-ca.grnspace.ca/wtc_signin.min.js"
```
Comment (ryanluker) [inline index.html:229]: @aemitos @evanwchurchill @matthewshirley I dont think we actualy updated this in cloudfront 🤔 .
I get nothing when I go to https://cdn-ca.grnspace.ca/wtc_signin.min.js but I still get something at https://cdn-ca.grnspace.co/wtc_signin.min.js

You can also see none of the demo site works atm either.
https://grnspace.github.io/wtc-oidc-client/

Comment (evanwchurchill) [reply index.html:229]: This was just merged this morning during standup

**Pull request merged**